package.json

```diff
@@ -1,6 +1,7 @@
 {
   "private": true,
   "scripts": {
+    "build-project": "yarn global add node-gyp && yarn --ignore-engines && yarn workspace @hicommonwealth/core build && yarn workspace commonwealth migrate-db",
```

Comment:
this is a temporary interface to build the project, used by CI in multiple places right now, but it will also be modified and included in the heroku scripts in a future PR implementing TS project references

Reply:
do we have documentation about composite action standards? cannot find anything under "composite action" in docs

Reply:
No documentation I'm just pointing out we should start using them if we have long repeated steps in CI. We should add documentation for this command in `Package-scripts.md` though.

Reply:
will do... although this will change soon